tito.spec

```diff
@@ -20,7 +20,7 @@ Version: 0.6.22
 Release: 1%{?dist}
 Summary: A tool for managing rpm based git projects
 
-License: GPLv2
+License: GPL-2.0-only
```

Comment:
Funny thing, the `setup.py` says `license='GPLv2+'`.
Should we change it? Or should we do `GPL-2.0-or-later` here?

Reply:
I think setup.py is newer. I would use the older one as a source of truth.